src/github_sdk.py

```diff
@@ -71,19 +71,29 @@ def send_envelope(trace):
 def get_extra_metadata(job):
     runs = get(job["run_url"]).json()
     workflow = get(runs["workflow_url"]).json()
+    repo = runs["head_repository"]["full_name"]
     meta = {
-        "workflow_name": workflow["name"],
-        # .github/workflows/foo.yml -> foo.yml
-        "workflow_path": workflow["path"].rsplit("/")[-1],
-        "head_branch": runs.get("head_branch"),
-        "head_sha": runs.get("head_sha"),
+        # "workflow_name": workflow["name"],
         "author": runs["head_commit"]["author"],
-        "run_attempt": runs["run_attempt"],
+        # https://getsentry.atlassian.net/browse/TET-22
+        # Tags are not linkified externally, plain text data can be selected in browsers and opened
+        "data": {
+            "job": job["html_url"],
+        },
+        "tags": {
+            "job_status": job["conclusion"],  # e.g. success, failure, skipped
+            "branch": runs["head_branch"],
+            "commit": runs["head_sha"],
+            "repo": repo,
+            "run_attempt": runs["run_attempt"],  # Rerunning a job
+            # It allows querying jobs within the same workflow (e.g. foo.yml)
+            "workflow": workflow["path"].rsplit("/")[-1],
+        },
     }
     if runs.get("pull_requests"):
-        meta[
-            "pull_request"
-        ] = f'https://github.com/{runs["head_repository"]["full_name"]}/pull/{runs["pull_requests"][0]["number"]}'
+        pr_number = runs["pull_requests"][0]["number"]
+        meta["data"]["pr"] = f"https://github.com/{repo}/pull/{pr_number}"
```

Comment:
This is new, it allows for selecting text in an issue and then being able to navigate to the PR.

For [instance](https://sentry.io/organizations/sentry-test/performance/armen-test-project:dfb378a450374d95853e546f94e88a8b/?project=6374757&query=transaction.duration%3A%3C15m&statsPeriod=24h&transaction=create+frontend+bundle&unselectedSeries=p100%28%29):
<img width="758" alt="image" src="https://user-images.githubusercontent.com/44410/167472954-47d47435-4aa4-4aba-904b-5be5b5b7bda5.png">